app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  validates_presence_of :content
+  validates_presence_of :upvotes
+
+  belongs_to :commentable, polymorphic: true 
```

Comment:
Love it. Polymorphic relationships are cool and were a great way to solve this problem.